website/docs/r/nsxt_edgegateway.html.markdown

```diff
@@ -0,0 +1,199 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_nsxt_edgegateway"
+sidebar_current: "docs-vcd-resource-nsxt-edge-gateway"
+description: |-
+  Provides a VMware Cloud Director NSX-T edge gateway. This can be used to create, update, and delete NSX-T edge gateways connected to external networks.
+---
+
+# vcd\_nsxt\_edgegateway
+
+Provides a VMware Cloud Director NSX-T edge gateway. This can be used to create, update, and delete NSX-T edge gateways connected
+to external networks.
+
+Supported in provider *v3.1+*
```

Comment:
Thinking out loud here, given that this is a first pure NSX-T resource, would there be a benefit to add from which VCD version it's supported?

Reply:
Potentially not. Good point.

Reply:
Removed.

Reply:
Added both version introduced and minimum versions required.